ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml

```diff
@@ -348,7 +348,7 @@ tests:
       resources:
         requests:
           cpu: 100m
-    workflow: ipi-azure
+    workflow: ipi-azure-ovn
```

Comment:
Job name should probably be called `e2e-kubevirt-azure-ovn`

Reply:
the job isn't related to kubevirt.

Reply:
fixed